packages/bitcore-node/src/models/transaction.ts

```diff
@@ -274,33 +278,67 @@ export class TransactionModel extends BaseTransaction<IBtcTransaction> {
     const rpc = new AsyncRPC(username, password, host, port);
     try {
       const result = await rpc.call('gettransactionreceipt', [txid]);
-      if (result.length > 0 && result[0].contractAddress) {
+      if (result.length > 0 && result[0].contractAddress && result[0].log.length === 0) {
         const contractAddress = result[0].contractAddress;
-        let name = '';
-        let symbol = '';
-        let totalSupply = 0;
         const decimals = await rpc.call('frc20decimals', [contractAddress]);
         if (decimals !== 0) {
-          name = await rpc.call('frc20name', [contractAddress]);
-          symbol = await rpc.call('frc20symbol', [contractAddress]);
-          totalSupply = await rpc.call('frc20totalsupply', [contractAddress]);
+          const name = await rpc.call('frc20name', [contractAddress]);
+          const symbol = await rpc.call('frc20symbol', [contractAddress]);
+          const totalSupply = await rpc.call('frc20totalsupply', [contractAddress]);
```

Comment:
we should call them all before consider this as a frc20

Reply:
ok kub